addon/styles/_frost-button.scss

```diff
@@ -305,6 +297,54 @@ $frost-color-button-in-line-darken-2: darken($frost-color-blue-1, 15);
       box-shadow: none;
     }
   }
+  
+  // design = 'app-bar'
+  &.app-bar {
```

Comment:
They have their own look and there are no effects for this kinds of button.
I thought it fit well for the design type as it did not have any other required parameters.